composer.json

```diff
@@ -17,30 +20,26 @@
     "license": "Apache-2.0",
     "minimum-stability": "stable",
     "require": {
-        "php": ">=7.1",
+        "php": ">=8.0",
```

Comment:
Surely this should be `~8.0 || ~8.1` - As it's not tested on 8.2?

Reply:
Your right! I guessed >= 8.0 should be good for now, but your version is obviously better in the future!

Reply:
Do you think we can drop this comment from the README File?

> Because of a dependency on the event dispatcher, this library is compatible with Symfony version 2.8 and higher. For Symfony 2.7 projects, use version 2.0.3.

The versions are extremely old and the event dispatcher got removed (as far as I am aware).

Reply:
> Do you think we can drop this comment from the README File?

Yes - it's clearly very outdated 👍

Reply:
I removed the notice in [8d788f7](https://github.com/prismicio-community/php-kit/pull/190/commits/8d788f7b034a9f2f79d65ba36b0769bbb970f3a7)

This is therefore resolved 👍🏼